app/assets/javascripts/discourse/components/user-card-contents.js.es6

```diff
@@ -53,6 +53,7 @@ export default Ember.Component.extend(
     showCheckEmail: Ember.computed.and("user.staged", "canCheckEmails"),
 
     user: null,
+    loading: false,
```

Comment:
Minor nit-pick: this line is not actually useful since `loading` is already defined in `CardContentsBase` mixin.

Reply:
Thanks, didn't know that :)